micromamba/src/common_options.cpp

```diff
@@ -256,6 +238,17 @@ override_channels_hook(bool& value)
             << "'override_channels' disabled by 'override_channels_enabled' set to 'false' (skipped)";
         value = false;
     }
+
+    if (value)
```

Comment:
Do we really need this? We actually do not have "`defaults`" at all in micromamba. If we have to add `nodefaults` here, it would mean that the `defaults` channel has been configured in the `.mambarc` file. However, that should already be taken care of by using the `--override-channels` option, right?

Reply:
Despite it's name `nodefaults` has nothing to do with `defaults`.

`defaults` tells conda to use the default anaconda channels. `nodefaults` tells conda to not use any channels from .condarc, but only use channels from the current environment.yml file.